### PR TITLE
Implement SSM instances and compliance items

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

**Comment by irmatov** [comment]
it's not clear how to do e2e for this, if at all possible

File: resources/ssm_instances.go [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+func SsmInstances() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_ssm_instances",
+		Description:  "Describes a filter for a specific list of instances.",
+		Resolver:     fetchSsmInstances,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "region", "instance_id"}},
+		Columns: []schema.Column{
```
Comment (roneli) [inline resources/ssm_instances.go:23]: can we add ARN for the resource https://docs.aws.amazon.com/service-authorization/latest/reference/reference_policies_actions-resources-contextkeys.html if it exists ?

File: resources/ssm_instances.go [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func SsmInstances() *schema.Table {
+	return &schema.Table{
```
Comment (roneli) [inline resources/ssm_instances.go:15]: would remove "Member is required." from descriptions

**Review by roneli — CHANGES_REQUESTED** [review]
LGTM, had one comment regarding adding an ARN.

File: resources/ssm_instances.go (hunk 2 above)
Comment (irmatov) [reply resources/ssm_instances.go:15]: Done

File: resources/ssm_instances.go (hunk 1 above)
Comment (irmatov) [reply resources/ssm_instances.go:23]: Added

File: resources/ssm_instances.go [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+				Name:        "computer_name",
+				Description: "The fully qualified host name of the managed instance.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "ip_address",
+				Description: "The IP address of the managed instance.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("IPAddress"),
+			},
```
Comment (roneli) [inline resources/ssm_instances.go:79]: Convert this to TypeInet

File: resources/ssm_instances.go [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+				Name:        "association_overview_detailed_status",
+				Description: "Detailed status information about the aggregated associations.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("AssociationOverview.DetailedStatus"),
+			},
+			{
+				Name:        "association_overview_instance_association_status_aggregated_count",
+				Description: "The number of associations for the instance(s).",
+				Type:        schema.TypeJSON,
+				Resolver:    schema.PathResolver("AssociationOverview.InstanceAssociationStatusAggregatedCount"),
```
Comment (roneli) [inline resources/ssm_instances.go:62]:
```suggestion
				Name:        "association_instance_status_aggregated_count",
				Description: "The number of associations for the instance(s).",
				Type:        schema.TypeJSON,
				Resolver:    schema.PathResolver("AssociationOverview.InstanceAssociationStatusAggregatedCount"),
```

**Review by roneli — APPROVED** [review]
LGTM, added a few more comments

File: resources/ssm_instances.go (hunk 3 above)
Comment (irmatov) [reply resources/ssm_instances.go:79]: Done

File: resources/ssm_instances.go (hunk 4 above)
Comment (irmatov) [reply resources/ssm_instances.go:62]: Done

**Pull request merged**